src/v1/spec.yaml

```diff
@@ -1286,7 +1286,7 @@ paths:
       tags:
         - Follows
       summary: Gets all followers for a given FID
-      description: Gets a list of users who follow a given user in reverse chronological order.
+      description: Gets a list of users who follow a user in reverse chronological order.
```

Comment:
`Followers`?

Reply:
that's the description, so shouldn't need to shorten it ig?